conf/yarn/reference.conf

```diff
@@ -38,6 +38,6 @@ gobblin.yarn.work.dir=/gobblin
 gobblin.cluster.helix.cluster.name=GobblinYarn
 gobblin.cluster.zk.connection.string="localhost:2181"
 
-fs.uri="hdfs://localhost:9000"
```

Comment:
Just curious, why port number needs to be changed here? Seems both 8020/9000 can be default port number for IPC of Namenode

Reply:
yes, both are acceptable, just changing it to 8020 as default which i believe most people use, can change it to 9000 if its otherwise, no prob.